tests/integration/integration_test.go

```diff
@@ -1253,6 +1265,9 @@ func TestResourceWithSecretSerialization(t *testing.T) {
 }
 
 func TestStackReferenceSecretsNodejs(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Temporarily skipping test on Windows")
```

Comment:
Minor - but could reference the issue number here.

Reply:
done!